src/components/forms/power-vs/PowerVsVolumeForm.js

```diff
@@ -76,7 +76,15 @@ class PowerVsVolumeForm extends React.Component {
       }
       nextState[name] = value;
       this.setState(nextState);
-    } else this.setState(this.eventTargetToNameAndValue(event));
+    } 
+    else if(name === "pi_storage_pool" || name === "pi_volume_pool") {
```

Comment:
This was added to handle the case when replication is enabled and set to true, and the pool is changed to one without replication enabled, and the toggle was still set to true. The state needs to be set in this way so that when the toggle should be disabled based on the selected pool, the toggle is also set to false.